src/libs/ajax/leonardo/Runtimes.ts

```diff
@@ -67,11 +67,12 @@ export const Runtimes = signal => {
         return res.json()
       },
 
-      create: (options): Promise<void> => {
+      create: (options, useExistingDisk: boolean = false): Promise<void> => {
```

Comment:
I can make a follow-up ticket to change that functionality, but it is _kinda_ a moot point because it'll error if there are multiple disks anyways, ie in the happy path there is only one possible disk to attach

Reply:
there could be a possible follow-up to allow selection from multiple PDs if that's something users want